apps/condo/domains/billing/schema/BillingAccount.test.js

```diff
@@ -510,4 +522,157 @@ describe('BillingAccount', () => {
             }, `${UNEQUAL_CONTEXT_ERROR}:property:context] Context is not equal to property.context`)
         })
     })
-})
\ No newline at end of file
+
+    describe('DiscoverServiceConsumers hook', () => {
+        let context
+        beforeAll(async ()=>{
+            const { context: billingContext }  = await makeContextWithOrganizationAndIntegrationAsAdmin(
```

Comment:
It's better not to create everything from admin, to be more close to real-life scenario...

Reply:
The acquiring context is not taken into account here? it is only needed for filtering receipts, right?